### PR TITLE
Adding varsome button

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

From @ninanorgren, see #1805 and #1860!

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by
- [x] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
Right, and I had like 2 mini-comments on this as well:
- In scout/server/blueprints/variant/utils.py, a ´genome_build(str)' is missing from the `update_transcripts_information(` doctstring. 
 - the class of the new links td needs an update: a weird additional internal boundary outline is rendered in the cells as is now

File: scout/server/blueprints/variant/templates/variant/tx_overview.html [hunk 1]
```diff
@@ -91,6 +94,12 @@
                       {{ (transcript.protein_sequence_name or '')|url_decode }}
                     </span>
                   </td>
+                  <td class="d-flex justify-content-center">
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/tx_overview.html:97]:
```suggestion
                  <td class="text-center">
```

Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/tx_overview.html:97]: This should fix the weird outline @dnil mentioned

File: scout/server/blueprints/variant/utils.py [hunk 2]
```diff
@@ -181,7 +181,7 @@ def add_gene_info(store, variant_obj, gene_panels=None, genome_build=None):
         panel_info = add_panel_specific_gene_info(extra_info.get(hgnc_id, []))
         variant_gene.update(panel_info)
 
-        update_transcripts_information(variant_gene, hgnc_gene, variant_obj)
+        update_transcripts_information(variant_gene, hgnc_gene, variant_obj, genome_build)
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/utils.py:185]: Could you add a docstring here? (original suggestion by @dnil)

File: scout/server/links.py [hunk 3]
```diff
@@ -266,6 +267,15 @@ def smart(smart_domain):
     return link.format(smart_domain)
 
 
+def varsome(build, refseq_id, protein_sequence_name):
+    if not all([refseq_id, protein_sequence_name]):
```
Comment (northwestwitch) [inline scout/server/links.py:271]: Could you add a docstring here as well?

File: scout/server/blueprints/variant/utils.py (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/variant/utils.py:185]: Ehm I don't think it was here that he meant!

Comment (dnil) [reply scout/server/blueprints/variant/utils.py:185]: Almost, it was for this function at least.. 😸 Look where it is defined!

Comment (northwestwitch) [reply scout/server/blueprints/variant/utils.py:188]: Yeah I fixed it I think!

**Comment by northwestwitch** [comment]
@ninanorgren I took the liberty of fixing your PR because we are preparing a new release of Scout and it would be nice to add this functionality in it, hope it's fine with you!

**Comment by northwestwitch** [comment]
I'm planning to merge this, anybody against speak now or never..

**Comment by dnil** [comment]
Just tested: looks ok.

**Comment by northwestwitch** [comment]
> Just tested: looks ok.

Thanks!

**Comment by ninanorgren** [comment]
Ah, see I was to late to do the fixes. Sorry, next time I'll be faster :) Thanks @northwestwitch for the fixes

**Comment by northwestwitch** [comment]
> next time I'll be faster

There's really no rush in general, we just wanted to wrap up everything for a new release today!

**Pull request merged**